cmd/mr_show.go

```diff
@@ -221,11 +236,11 @@ func printMRDiscussions(discussions []*gitlab.Discussion, since string, mrNum in
 				printit = color.New(color.Bold).PrintfFunc()
 			}
 			printit(`
-%s%s %s at %s
+#%d: %s%s %s at %s
```

Comment:
Sigh, would love for these to eventually get updated to be based off `text/template`s, this originally came in as a PR and I've never gotten around to cleaning it up. IIRC the issue/mr/snippet create commands use templates, should you find yourself inspired to clean this up.

Not blocking / unrelated to this PR tho.

Reply:
I'll go through and take a look at using text/template.  If you have the branch lying around let me know and I'll take a look.

Reply:
This is the best reference in the repo most likely https://github.com/zaquestion/lab/blob/fea5044402775b8777520f7dfcc03d0804a6ba1b/cmd/mr_create.go#L246-L295